docs/getting-started/javascript-options-api.md

```diff
@@ -8,253 +8,163 @@ heading: Get Started
 position: 10
 ---
 
-# Get Started with Kendo UI for Vue Native Components with JavaScript and Options API
+# Get Started with Kendo UI for Vue
 
-In this article you’ll learn how to use Kendo UI for Vue components by building a small app that includes a Grid, a DropDownList, a Window and a design theme.
+This tutorial will help you develop a simple app that includes a native Vue Data Grid component. To achieve this, you will build a project using [Vite](https://vitejs.dev/) and the [Vue Options API](https://vuejs.org/guide/typescript/options-api) paired with JavaScript.
+
+
+>Curious about TypeScript or the Composition API? This tutorial comes in several additional variants:
+>* [Kendo UI for Vue with TypeScript and the Composition API](slug:getting_started_typescript_composition_api)
+>* [Kendo UI for Vue with TypeScript and the Options API](slug:getting_started_typescript_options_api)
 
-The current example uses the recommended by Vue [Vite build tool](https://vitejs.dev/) + [Vue Options API](https://vuejs.org/guide/introduction.html#options-api). If you need Getting started information that uses `Vue Composition API`, you can check [this link](slug:getting_started_javascript_composition_api).
 
 > Historically, all Kendo UI for Vue Native components support both **Vue 2** and **Vue 3**, however, Vue 2 is currently in its end-of-support phase till Nov 2024. After our last major release for 2024, Vue 2 will no longer be supported in the new versions of the Kendo UI for Vue components. Please check our [Vue 2 End of Support article](https://www.telerik.com/kendo-vue-ui/components/vue2-deprecation/) for more details.
 
-## 1. Set up the Vue project
-* The recommended way to start with Vue is to scaffold a project using [Vite](https://vuejs.org/guide/scaling-up/tooling.html#vite). To create a new Vite project use one of the following commands for NPM or Yarn.
+## Create the Vue Project
 
-Create project commands:
+The recommended way to scaffold your Vue project is using [Vite](https://vuejs.org/guide/scaling-up/tooling.html#vite).
 
-```sh
-npm create vite@latest
-```
-or
-```sh
-yarn create vite
-```
 
-When one of the above commands is executed, the interface will ask you to apply additional configurations to your project:
-1. Set the project name: 
-Here we can define the name of our new project. For the needs of the current article, set the name of the application as `my-app`
+> You can use both NPM and Yarn to create the project and import the Kendo UI for Vue components. However, for the purposes of this tutorial, we demonstrate only NPM.
```

Comment:
I know this is what Dancho suggested here. However, our style guide recommends against using "we". Consider the following statement instead:
```suggestion
> You can use both NPM and Yarn to create the project and import the Kendo UI for Vue components. This tutorial demonstrates only the NPM approach.
```